sonar-scanner-engine/src/main/java/org/sonar/scanner/task/ScanTask.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.sonar.scanner.task;
 
+import org.sonar.scanner.scan.BranchConfigurationProvider;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonar.scanner.scan.BranchConfigurationProvider'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)